### ISSUE TITLE
[Bug]: Select widget's scrollbar acts differently from the other select widgets

### ISSUE THREAD

**Issue by momcilo-appsmith** [appsmithorg/appsmith#9550]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

here is a space that is created for a scrollbar in the select widget that stays permanently there.  It happens only for the select widget and other select widgets (tree, multi, multi-tree) don’t have this problem. 
![image](https://user-images.githubusercontent.com/81744497/144566393-e14289a6-dabb-47b7-9fa0-0ad148febe31.png)

### Steps To Reproduce

1. Add a select widget
2. Add 6-7 items to it
3. Check the scrollbar

### Environment

Production

### Version

Cloud

**Comment by techbhavin** [comment]
In Select Widget 
-  Custom scroll Design is applied from `app/client/src/assets/styles/index.css`

But For Other Widget Like multi/tree select Widget 
-   Here we are using the `rc-select / rc-tree-select` package and they using rc-virtual-list underhood and manage scrollbar by itself because of it they are showing different scroll bars.

Here we Don’t have any control to customize the behavior of `rc-select / rc-tree-select`.  

cc: @somangshu  @sbalaji1192  @Tooluloope

**Comment by sbalaji1192** [comment]
@Tooluloope Do we have plans to rewrite the select widget with `rc-select`?

**Comment by Tooluloope** [comment]
@sbalaji1192 Yes, tracking the requirements here https://github.com/appsmithorg/appsmith/issues/10575.

Although, My reason for delaying is rc-select and rc-tree-select have difficulty when dragging the input field as seen here https://github.com/appsmithorg/appsmith/issues/7259

**Comment by somangshu** [comment]
I would be for that @sbalaji1192, I hope we have thought this through and do not have to fallback to another packages for any any probably issue in the future.

**Comment by sbalaji1192** [comment]
@somangshu Sure.
@Tooluloope Let's connect before proceeding with https://github.com/appsmithorg/appsmith/issues/10575

**Comment by dilippitchika** [comment]
## Stats
| Stat | Values |
| ------------- | ------------- |
| Reach     | 500 |
| Effort (months)  | 0.5 |
